src/topp/FeatureLinkerBase.cpp

```diff
@@ -44,6 +44,8 @@
 
 #include <OpenMS/KERNEL/ConversionHelper.h>
 
+#include <OpenMS/SYSTEM/SysInfo.h>
+
 #include <OpenMS/APPLICATIONS/TOPPBase.h>
 
 #include <iomanip>     // setw
```

Comment:
ALphabetical order of the includes, also see line 35 - 38

Reply:
we're on it